include/swift/Sema/CSFix.h

```diff
@@ -295,6 +295,13 @@ enum class FixKind : uint8_t {
   /// Ignore result builder body if it has `return` statements.
   IgnoreResultBuilderWithReturnStmts,
 
+  /// Ignore `ErrorExpr` or `ErrorType` during pre-check.
+  IgnoreInvalidASTNode,
+
+  /// Ignore the a named pattern whose type we couldn't infer. This issue should
```

Comment:
"the" is extraneous here? 
```suggestion
  /// Ignore a named pattern whose type we couldn't infer. This issue should
```